doc/release-notes.rst

```diff
@@ -2,6 +2,104 @@
  Release Notes
 ===============
 
+v12.2.1 Luminous
+================
+
+This is the first bugfix release of Luminous v12.2.x long term stable
+release series. It contains a range of bug fixes and a few features
+across CephFS, RBD & RGW. We recommend all the users of 12.2.x series
+update.
+
+Notable Changes
+---------------
+
+* Dynamic resharding is now enabled by default for RGW, RGW will now
+  automatically reshard there bucket index once the index grows beyond
+  `rgw_max_objs_per_shard`
+
+* Limiting MDS cache via a memory limit is now supported using the new
+  mds_cache_memory_limit config option (1GB by default).  A cache reservation
+  can also be specified using mds_cache_reservation as a percentage of the
+  limit (5% by default). Limits by inode count are still supported using
+  mds_cache_size. Setting mds_cache_size to 0 (the default) disables the
+  inode limit.
+
+* The maximum number of PGs per OSD before the monitor issues a
+  warning has been reduced from 300 to 200 PGs.  200 is still twice
+  the generally recommended target of 100 PGs per OSD.  This limit can
+  be adjusted via the ``mon_max_pg_per_osd`` option on the
+  monitors.  The older ``mon_pg_warn_max_per_osd`` option has been removed.
+* Creating pools or adjusting pg_num will now fail if the change would
+  make the number of PGs per OSD exceed the configured
+  ``mon_max_pg_per_osd`` limit.  The option can be adjusted if it
+  is really necessary to create a pool with more PGs.
+
+* There was a bug in the PG mapping behavior of the new *upmap*
+  feature. If you made use of this feature (e.g., via the `ceph osd
+  pg-upmap-items` command), we recommend that all mappings be removed (via
+  the `ceph osd rm-pg-upmap-items` command) before upgrading to this
+  point release.
+
```

Comment:
* A stall in BlueStore IO submission that was affecting many users has been resolved.

Reply:
ack